ios/PlayerModule.swift

```diff
@@ -347,6 +387,31 @@ class PlayerModule: NSObject, RCTBridgeModule {
         }
     }
 
+    /**
+     Set `nativeId`'s player subtitle track.
+     - Parameter nativeId: Target player Id.
+     - Parameter trackIdentifier: The subtitle track identifier.
+     - Parameter resolver: JS promise resolver.
+     - Parameter rejecter: JS promise rejecter.
+     */
+    @objc(setSubtitleTrack:trackIdentifier:resolver:rejecter:)
+    func setSubtitleTrack(
+        _ nativeId: NativeId,
+        trackIdentifier: String,
```

Comment:
Shouldn't `trackIdentifier` be optional? It is optional for for Android in `PlayerModule.kt`.
In that case the `if` statement can be replaced with a simple `self?.players[nativeId]?.setSubtitle(trackIdentifier: trackIdentifier)` call.